app/views/ratings/create.turbo_stream.erb

```diff
@@ -0,0 +1,12 @@
+<%= turbo_stream.update(
+      "movie_rated_#{@movie.tmdb_id}",
+      partial: 'movies/movie_rating',
+      locals: { movie: @movie }
+    )
+%>
+<%= turbo_stream.update(
+      "movie_show_rated_#{@movie.tmdb_id}",
+      partial: 'movies/movie_rating',
+      locals: { movie: @movie }
+    )
+%>
```

Comment:
This is a result of us having complexity around showing things in the modal vs show page, for example